libraries/program-error/derive/src/macro_impl.rs

```diff
@@ -11,7 +11,7 @@ use {
 };
 
 const SPL_ERROR_HASH_NAMESPACE: &str = "spl_program_error";
-const SPL_ERROR_HASH_MIN_VALUE: u32 = 10_0000;
+const SPL_ERROR_HASH_MIN_VALUE: u32 = 7_0000;
```

Comment:
Haha I just realized that this has an extra 0, should be `7_000`. Good thing we have multiple rounds of reviews

Reply:
Ah! my bad